app/src/main/kotlin/com/wire/android/ui/common/avatar/UserProfileAvatar.kt

```diff
@@ -201,6 +210,18 @@ fun UserProfileAvatar(
             val exactPointOnAvatarBorder = sqrt(2f) / 2f * ((size.value / 2f) + avatarBorderWidth.value)
             val maxOffset = (size.value / 2f) - (statusSize.value / 2f) - -statusBorderWidth.value
             val offsetToAlignWithAvatar = min(maxOffset, exactPointOnAvatarBorder)
+
+            if (shouldShowCreateTeamUnreadIndicator) {
+                UnreadInfoIndicator(
+                    modifier = Modifier
+                        .offset(x = offsetToAlignWithAvatar.dp, y = -offsetToAlignWithAvatar.dp)
+                        .onGloballyPositioned {
+                            userStatusIndicatorParams = it.size.toSize() to it.positionInParent()
```

Comment:
it would be better to create separate param for "create team unread" indicator and not use the same as for "user status" because both have slightly different sizes and updating the same param from two different composables can make it so that position of one of them becomes wrong in the end.